src/odin_devices/si5324.py

```diff
@@ -0,0 +1,539 @@
+"""
+SI5324 - device access class for the SI5324 Clock Multiplier
+
+Class to drive the SI5324 Clock Multiplier IC. Settings should primarily
+be created using the DSPLLsim software for PC, which can be uploaded using
+the apply_register_map() function. From there, some settings can be tweaked
+and the input clock can be switched (see Manual Access Functions). Be sure
+to run calibrate() after settings are changed.
+
+Once the settings are as desired, a register map can be exported with
+export_register_map() for later use.
+
+Joseph Nobes, STFC Detector Systems Software Group
+"""
+
+from odin_devices.i2c_device import I2CDevice, I2CException
+import time
+import logging
+
+#logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.INFO)
```

Comment:
If you define a custom logger for a module and set its logging level (or adjust the root logger level) in source, you have no runtime control over the logging level. This will affect the level in e.g. the entire odin-control instance. We haven't addressed this properly in other modules/packages so it bears thinking about a bit....

Reply:
In that case, should I just be removing the logging level line completely so that it can be defined in the calling script?

Reply:
Yes, don't set the logging level explicitly in the module.

Reply:
Is the method of creating and using the logger otherwise correct? I've not used the module before, so I'm not sure how it should properly fit into a larger project.